src/ToggleBadge.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { theme as themeGet, space } from 'styled-system'
+import theme from './theme'
+
+const ToggleBadge = styled.div`
+  border-radius: 2px;
+  display: inline-block;
+  font-size: ${props => props.theme.fontSizes[0]}px;
+  font-weight: 600;
+  background-color: ${props =>
```

Comment:
not sure if this was already discussed with the design team but should the color/bg-color be configurable?

Reply:
Best approach here may be to allow any color with default props as lightBlue/darkBlue